.rubocop.yml

```
@@ -1,43 +1,27 @@
-require:
-  - rubocop-performance
-  - rubocop-rails
-  - rubocop-minitest
-
-# TODO: remove this file by either moving cops here or fixing code
 inherit_from:
   - .rubocop_todo.yml
 
+inherit_gem:
+  theforeman-rubocop:
+    - rules/base.yml
+    - rules/ruby_target.yml
+    - rules/style.yml
+    - rules/metrics.yml
+    - rules/performance.yml
+    - rules/rails.yml
+    - rules/minitest.yml
+    - rules/style_lenient.yml
```

Like https://github.com/theforeman/foreman-tasks/pull/750#discussion_r1549323058 I'd prefer to use the `lenient.yml` ruleset instead.
```suggestion
    - lenient.yml
```